joint_names: [left_arm_1_joint, left_arm_2_joint, left_arm_3_joint, left_arm_4_joint, left_arm_5_joint, left_arm_6_joint, left_arm_7_joint, right_arm_1_joint, right_arm_2_joint, right_arm_3_joint, right_arm_4_joint, right_arm_5_joint, right_arm_6_joint, right_arm_7_joint]
joint_initial_values: [-0.007011466954600481, -0.6805500000000102, -0.00349, -1.0210200000000031, -0.0026179999999999992, -1.0210200000000031, 0.0, 0.001745, 0.6665900000000093, 0.0, 0.8427231092462678, -0.0026179999999999992, 0.7304220000000022, 0.0 ]
Sampling_peroid: 0.01
frequency: 100
min_publish_duration: 0.01
